apps/src/blocklyAddons/functionEditor.js

```diff
@@ -0,0 +1,27 @@
+export default class FunctionEditor {
+  // TODO
+  constructor(
+    opt_msgOverrides,
+    opt_definitionBlockType,
+    opt_parameterBlockTypes,
+    opt_disableParamEditing,
+    opt_paramTypes
+  ) {}
+
+  // TODO
+  isOpen() {
+    return false;
+  }
+
+  // TODO
+  renameParameter(oldName, newName) {}
+
+  // TODO
+  refreshParamsEverywhere() {}
+
+  // TODO
+  openEditorForFunction(procedureBlock, functionName) {}
+
+  // TODO
```

Comment:
Not sure what works for your workflow, but do you have a task or reminder for at the 'end' of getting Outbreak onto Blockly to go back and make sure that all the 'ToDos' are addressed?

Reply:
Not all of these TODOs need to be addressed before HOC. In fact, I think most of them will __not__ be addressed as part of Outbreak work. The stubs will need to be filled in when we migrate all of Sprite Lab, but leaving them as no-op stubs for now is fine because Outbreak doesn't actually use the features

Reply:
I would argue that we should add additional details to the ToDos then - to indicate the lifespan on the ToDo and differentiate between ToDos that require action and expected No-Op functions.